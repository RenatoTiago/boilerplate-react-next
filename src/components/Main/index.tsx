import * as S from './styles'

const Main = () => (
  <S.Wrapper>
    <S.Logo src="img/react-icon.svg" alt="Image descrito Next JS" />
    <S.Title> Boilerplate Front-End</S.Title>
    <S.Description>
      {' '}
      React JS, Nexts JS, Styled components, TypeScript,{' '}
    </S.Description>
    <S.Illustration src="img/developer.svg" />
  </S.Wrapper>
)

export default Main
