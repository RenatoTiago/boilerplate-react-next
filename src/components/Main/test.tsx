import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'

import Main from '.'

describe('<Main/>', () => {
  it('should render the main', () => {
    const { container } = render(<Main />)
    expect(
      screen.getByRole('heading', { name: /Boilerplate/i })
    ).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
