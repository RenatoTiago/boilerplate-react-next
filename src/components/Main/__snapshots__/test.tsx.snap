// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render the main 1`] = `
<main
  class="styles__Wrapper-gVBhw iziTas"
>
  <img
    alt="Image descrito Next JS"
    class="styles__Logo-bSiBeQ wldvi"
    src="img/react-icon.svg"
  />
  <h1
    class="styles__Title-cMjpFf fxohpK"
  >
     Boilerplate Front-End
  </h1>
  <h2
    class="styles__Description-elAaux hrwMlQ"
  >
     
    React JS, Nexts JS, Styled components, TypeScript,
     
  </h2>
  <img
    class="styles__Illustration-fySXif kkSSNd"
    src="img/developer.svg"
  />
</main>
`;
